Octokit.Reactive/Clients/IObservableOrganizationsClient.cs

```diff
@@ -16,6 +16,12 @@ public interface IObservableOrganizationsClient
         IObservableTeamsClient Team { get; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="http://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        IObservableOrganizationHooksClient Hooks { get; }
```

Comment:
I added these properties based on the iobservablerepositoriesclient, it was 'Hooks' there. I thought that is how it should be named here too. I'll change it if you want